tools/qontract_cli.py

```diff
@@ -509,99 +509,10 @@ def cluster_upgrade_policies(
     show_only_soaking_upgrades=False,
     by_workload=False,
 ):
-    from reconcile.aus.ocm_upgrade_scheduler import (
-        OCMClusterUpgradeSchedulerIntegration,
-    )
-
-    integration = OCMClusterUpgradeSchedulerIntegration(
-        AdvancedUpgradeSchedulerBaseIntegrationParams()
-    )
-    generate_cluster_upgrade_policies_report(
-        ctx,
-        integration=integration,
-        cluster=cluster,
-        workload=workload,
-        show_only_soaking_upgrades=show_only_soaking_upgrades,
-        by_workload=by_workload,
-    )
-
-
-def generate_cluster_upgrade_policies_report(
-    ctx,
-    integration: AdvancedUpgradeSchedulerBaseIntegration,
-    cluster: Optional[str],
-    workload: Optional[str],
-    show_only_soaking_upgrades: bool,
-    by_workload: bool,
-) -> None:
-    md_output = ctx.obj["options"]["output"] == "md"
-
-    org_upgrade_specs = []
-    for orgs in integration.get_upgrade_specs().values():
-        for org_spec in orgs.values():
-            filtered_specs = org_spec.specs
-            if cluster:
-                filtered_specs = [s for s in filtered_specs if cluster == s.name]
-            if workload:
-                filtered_specs = [
-                    s for s in filtered_specs if workload in s.upgrade_policy.workloads
-                ]
-            if filtered_specs:
-                org_upgrade_specs.append(
-                    OrganizationUpgradeSpec(org=org_spec.org, specs=filtered_specs)
-                )
-
-    results = get_upgrade_policies_data(
-        org_upgrade_specs,
-        md_output,
-        integration.name,
-        workload,
-        show_only_soaking_upgrades,
-        by_workload,
+    print(
+        "https://grafana.app-sre.devshift.net/d/ukLXCSwVz/aus-cluster-upgrade-overview"
```

Comment:
I would rather keep the cli command as is, remove it from the app-interface-output jobs and set this URL in the markdown manually

Reply:
the report is not going to yield meaningful results anymore because the state for `ocm-upgrade-scheduler` (version history) will not change anymore because the state for these clusters is managed by `advanced-upgrade-service`

Reply:
i can at least keep the cli command for AUS and adapt the markdown manually as you suggested